docs/manual/pubsub.md

```diff
@@ -38,8 +38,9 @@ stream of messages where the first element indicates the type of
 message. The commands that are allowed in the context of a subscribed
 client are `SUBSCRIBE`, `SSUBSCRIBE`, `SUNSUBSCRIBE`, `PSUBSCRIBE`, `UNSUBSCRIBE`, `PUNSUBSCRIBE`, `PING`, `RESET`, and `QUIT`.
 
-Please note that `redis-cli` will not accept any commands once in
-subscribed mode and can only quit the mode with `Ctrl-C`.
+Please note that when a client using `redis-cli` is in subscription mode, 
```

Comment:
Sorry, there is still something in this sentence that seems odd.

* "a client using redis-cli" is strange because redis-cli is a client.
* The grammar "when using redis-cli is in ..." seems wrong. Better without "is" and with a comma instead.

```suggestion
Please note that when using `redis-cli`, in subscribed mode
```

Reply:
In sentences, I think it's better without "is" and with a ',' comma instead.
And I agree to exclude 'a client' as well.
@zuiderkwast, Thank you for your detailed review. :)